src/constants.ts

```diff
@@ -1,28 +1,28 @@
-import { NodeHelper } from "./helpers/NodeHelper"
-import { EnvHelper } from "./helpers/Environment"
-import ethereum from "./assets/tokens/wETH.svg"
-import arbitrum from "./assets/arbitrum.png"
-import avalanche from "./assets/tokens/AVAX.svg"
+import { NodeHelper } from "./helpers/NodeHelper";
+import { EnvHelper } from "./helpers/Environment";
+import ethereum from "./assets/tokens/wETH.svg";
+import arbitrum from "./assets/arbitrum.png";
+import avalanche from "./assets/tokens/AVAX.svg";
 
-export const THE_GRAPH_URL = "https://api.thegraph.com/subgraphs/name/drondin/olympus-graph"
-export const EPOCH_INTERVAL = 2200
+export const THE_GRAPH_URL = "https://api.thegraph.com/subgraphs/name/drondin/olympus-graph";
+export const EPOCH_INTERVAL = 2200;
```

Comment:
@0xtaichi we are going to use block timestamp to measure an epoch's duration instead of block numbers because Fantom's block time is not regular like Ethereum